src/integrationTest/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/CreateInductionTest.kt

```diff
@@ -109,31 +108,32 @@ class CreateInductionTest : IntegrationTestBase() {
   }
 
   @Test
-  fun `should fail to create induction given database exception`() {
+  fun `should create induction but fail to create timeline event given timeline database exception`() {
```

Comment:
The fact I've had to change this test proves that the previous implementation wasn't async at all!

The basis of this test is that a database exception is thrown whilst writing the timeline event, where the database exception is caused by using an invalid prison ID which is longer than the column width (yes, I know we should be doing request body validation ..... )

Anyway, the timeline event is recorded by the "async" timeline event service

Previously the entire request was failed because the timeline event service failed (with the database exception), and that exception was propagated all the way back to the controller and the global exception handler.
This proves that the timeline event service was never actually async!

Now that the timeline service is async the test has to change because we now expect the main request to succeed, and the timeline event (which is a "nice to have", but not a core part of the request) should silently fail.